src/confdb/confdb.c

```diff
@@ -1010,10 +1010,6 @@ static int confdb_get_domain_internal(struct confdb_ctx *cdb,
 
     if (!domain->enumerate) {
         DEBUG(SSSDBG_TRACE_FUNC, "No enumeration for [%s]!\n", domain->name);
-        DEBUG(SSSDBG_CONF_SETTINGS,
-              "Please note that when enumeration is disabled `getent "
-              "passwd` does not return all users by design. See "
-              "sssd.conf man page for more detailed information\n");
```

Comment:
I think this warning still has a value, I wouldn't remove it.

Reply:
Oops, sorry, you added the warning in the previous patch. So I guess don't include the first patch or just squash the two togetherin your branch.